Requirements for SSE.md

```diff
@@ -37,22 +37,21 @@ TODO: Insert Diagram photo
 
-#### TODO: Misuse case Name
+#### Updating Configuration.yaml Misuse Case
 
-TODO: Insert info here
+Bobs neighbor, Kyle, comes over to Bobs house for a weekend BBQ with family and friends. Bob, being the not so tech savvy user, gives out his WiFi password to the party goers so they can use their mobile devices. While Kyle helped Bob get started on HomeAssistant, Kyle is experienced in IT and HomeAssistant and knows the ins and outs of the software. He connects to Bob's WiFi network, saving the password on his phone. When Kyle gets home, he gets on his laptop and connects to Bob's WiFi from his computer. Kyle knows that Bob just set up his HomeAssistant so he begins to try and connect to the device and see Bob's Configuration.yaml file. First Kyle tried connecting to localhost:8123 but noticed that it required a username and password preventing him from viewing the Configuration.yaml with the File Editor addon. Kyle then begins to see if Bob enabled his device to allow editing via Samba. Sure enough Bob enabled Samba and Kyle sees the Configuration.yaml file under his Network tab in file explorer. Kyle attempts to open the Configuration.yaml file and notices that all of Bob's passwords are listed as !secrets <AccountName>. Kyle then looks at the secrets.yaml file which is in the same folder as the Configuration.yaml file and retrieves all of Bob's sensitive information. 
 
 #### Prevention/Security Requirement
 
-TODO: Assess alignment of security requirements derived from mis-use case analysis with advertised features of the open-source software. Review OSS project documentation and codebase to support your observations. Summarize your observations.
+To prevent the mis-use case, HomeAssisnt employs several safety measures. To prevent an attacker from gaining access to the Configurations.yaml file, the web interface is protected by a password and multi-factor authentication to prevent bruteforce attacks. In the event that a user will enable the Samba addon, Samba requires password authentication and the attacking machine to be on the same local network. In the event that a malicious actor does gain access to the Configuration.yaml, HomeAssistant documentation provides users the ability to input sensitive data in a separate yaml file known as the secrets.yaml file. This will allow the user to input !secrets <AccountName> in place of the sensitive data and the Configuration.yaml file will pull the associated information based on the <Accountname>. However the secrets.yamls file still stores information in plain text. HomeAssistant also gives the user the ability to store sensitive data in the systems Environmental Variables, storing passwords in an Amazon Web Services (AWS) server via [Credstash](https://github.com/fugue/credstash) or on a [keyring](https://github.com/jaraco/keyring) stored on the Operating System(OS). 
```

Comment:
Since Bob was giving out his main network password maybe he can set up a guest network to prevent this. Maybe this is out of scope. If so, please ignore.

Reply:
Yeah, a guest network is not something offered by HA to prevent the misuse case. I also did not see anything about that in the documentation or any sort of "network segmentation" so I'm thinking it is out of scope for this assignment. Maybe something we could contribute to in their documentation at a later point.